### DISCUSSION TITLE
Tagging SSWG packages

### DISCUSSION THREAD

**Discussion by adam-fowler** [SwiftPackageIndex/SwiftPackageIndex-Server#2548] [Ideas]
Hi,

We have been discussing in the SSWG meetings ways to make the packages in the SSWG incubation process more visible. One of these was to have some way of tagging them in the Swift Package Index. To give users of the index some guarantees about the packages they are investigating. Maybe also show a list of the relevant packages when you search for "SSWG".

What are your thoughts?
I just had a search for related topics and see this has been brought up before, by @mattpolzin in #1352.

**Comment by finestructure** [comment]
Thanks for raising this, @adam-fowler , I've long wanted to have something like this on SPI.

If this is purely about having a way to discover SSWG package, simply assigning keywords on Github would make them searchable on SPI. That's a start but only minimally useful, I suppose.

For one, anyone could opt into that tag by assigning it to their package. It also wouldn't expose a package collection that developers could import into Xcode.

What exactly are the features you're looking for? I presume, as mentioned,

- search
- package collection

Would there be "tags" for the different incubation stages? Do you envision some sort of badge/label on the package page for packages in certain stages?
